components/Card.js

```diff
@@ -1,7 +1,44 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
+const Block = ({ month, day, year, type, time, place, description }) => {
+	return (
+		// have one shadow white and one black right behind it looking like a border
+
+		<Card
+			style={{
+				width: "15rem",
+				backgroundColor: "black",
+				borderRadius: 25,
+				boxShadow: "12px -13px white, 16px -17px black",
+			}}
+		>
+			<Card.Body>
+				<div className='flex justify-evenly '>
+					<div className='columns-1 text-blue-500'>
+						<Card.Title>{month}</Card.Title>
```

Comment:
Why not shove it into one like? Or does it not show up on new lines? You can use `<br/>` to make that happen

```Javascript
<Card.Title>{month} {day} {year}</Card.Title>

```